release/release_logs/2.3.0/scalability/single_node.json

```diff
@@ -1,8 +1,8 @@
 {
-    "args_time": 16.49447203899996,
-    "get_time": 24.665624744000013,
+    "args_time": 16.44382056200004,
+    "get_time": 24.781213436999963,
     "large_object_size": 107374182400,
-    "large_object_time": 325.56598805499993,
+    "large_object_time": 490.7879660510001,
```

Comment:
is this expected?

Reply:
https://docs.google.com/spreadsheets/d/1_rPUOGDNq-Q_nHt42iNF8C8XF55cyMOaz6B0k87fsEA/edit#gid=0

Reply:
I think this is the one with very high variance on the V1 stack due to EBS. I think we should run it again to be sure that it's a regression and not just network tail latency.

https://github.com/ray-project/ray/issues/30711#issuecomment-1405956921

Reply:
Synced with @rickyyx @cadedaniel, the metrics difference is believed to just be the expected variance. Kicked off a rerun of tests `single_node` and `microbenchmark` [here](https://buildkite.com/ray-project/release-tests-branch/builds/1438) so that we can be more confident in this hypothesis. @scv119

Reply:
thanks @zcin, yeah let's retry here!

Reply:
Seems like this may not be a transient issue...the rerun results are uploaded at [this PR](https://github.com/ray-project/ray/pull/33197) for ease of comparing. 

- New results for `large_object_time` are [here](https://github.com/ray-project/ray/pull/33197/files#diff-656f13e098270dac02931057378e430f823d210eba57f3eff0361cc39f4cb944L5-R5), still a 49% regression.
- New results for `1_n_async_actor_calls_async` are [here](https://github.com/ray-project/ray/pull/33197/files#diff-222cd83bbbc9457757b3aafe7ba4440c644bbcf9dba16c0ddb6ce50e801d54e2L31-R32), <1%.